lepton/starterkits/model_compression.ipynb

```diff
@@ -0,0 +1,701 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6ebbe7d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!accelerate launch -m lm_eval --model hf --model_args pretrained=\"{DEPTH_PRUNED_MODEL_DIR}/checkpoints/best_hf\" --batch_size 4 --seed 1234 --tasks mmlu --num_fewshot 5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16035412",
+   "metadata": {},
+   "source": [
+    "Here is a summary of the results on the small Wikitext dataset (only ~125M tokens):\n",
+    "\n",
+    "| Model | MMLU |\n",
+    "|-------|------|\n",
+    "| Qwen3-8B | 74.9 |\n",
+    "| Depth-Pruned 6B | 62.6 |\n",
+    "| Width-Pruned 6B | 56.4 |\n",
+    "| Qwen3-4B | 70.0 |\n",
+    "\n",
+    "> **NOTE:** The dataset used here is fairly small so the results are not very conclusive. In practice with larger datasets, width pruned models have higher MMLU scores but depth pruned models are faster at inference at the same number of parameters. The difference in accuracy narrows on using better quality datasets for longer distillation.\n",
+    "\n",
+    "#### Importance of Dataset Quality\n",
+    "\n",
+    "If instead of Wikitext dataset, we used better datasets like [ClimbMix](https://huggingface.co/datasets/OptimalScale/ClimbMix) or [Nemotron-Pretraining-SFT-v1](https://huggingface.co/datasets/nvidia/Nemotron-Pretraining-SFT-v1), MMLU for the depth pruned 6B model would be around **72.5** and **73** respectively. Width pruned model could be even higher. Here the distillation is performed for ~6k H100 GPU hours (96 nodes with 8 H100 each * 8 hours) using **~90B tokens**. Further distillation on these datasets could yield even better results. The Nemotron-Pretraining-SFT-v1 dataset also has good quality coding, multilingual and other task data hence would also result in improvement on other pre-training benchmarks apart from MMLU."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e9e49e66",
+   "metadata": {},
+   "source": [
+    "### Next Steps\n",
+    "\n",
+    "So far, we have distilled the pruned models on a pre-training dataset hence the model is a base variant. Since we have a base model, we only compared all the models on base model benchmarks like MMLU. To practically use these models for reasoning tasks, we need to perform post-training on these models as well which is something we will add to this tutorial in the near future.\n",
+    "\n",
+    "We can also further Quantize these models to FP8 precision using [TensorRT Model Optimizer](https://github.com/NVIDIA/TensorRT-Model-Optimizer/tree/main/examples/llm_ptq) and measure Tokens per Second (TPS) for inference. We observed that the depth pruned 6B model is ~30% faster than the Qwen3-4B and ~60% faster than the Qwen3-8B when all are quantized to FP8 precision on single H100 GPU."
+   ]
+  }
+ ],
+ "metadata": {
+  "language_info": {
+   "name": "python"
+  }
```

Comment:
Can you add the following metadata tags to this section?
1. `"container_image": "<the container image:tag this should run on>"`
2. `"title": "<Title for the notebook, ie. Model compression with NeMo>"`
3. `"description": "<One-sentence description of the notebook>"`